src/1_utils.js

```diff
@@ -263,3 +263,20 @@ utils.base64encode = function(input) {
 	}
 	return output;
 };
+
+/**
+ * Extract the deeplink path from urls in the forms of:
+ * - "protocol://domain.name/deeplink.path
+ * - "domain.name/deeplink.path"
```

Comment:
i think qinwei is correct, though, if you're going for the logic in that comment, you would only return `deeplink.path` - i think that's what this function does, but the comment doesn't make that clear

that part of the url is usually called the "path"